tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -435,173 +375,138 @@ describe(__filename, () => {
+  // });
+
+  // it('renders AddonsByAuthorsCard for extensions', () => {
+  //   const userId = 123;
+  //   const { params, store } = signInUserWithUserId(userId);
+
+  //   const { getByText } = renderUserProfile({ params, store });
+
+  //   expect(screen.getByText('Extensions by')).toBeInTheDocument();
+  //   const extension_card = screen.queryByClassName('AddonsByAuthorsCard').at(0);
+  //   expect(getByText(extension_card, 'Previous')).toBeInTheDocument();
+  //   expect(getByText(extension_card, 'Next')).toBeInTheDocument();
+  //   expect(
+  //     getByText(extension_card, `/user/${userId}/?page_e=2`),
+  //   ).toBeInTheDocument();
+  // });
+
+  // it('renders AddonsByAuthorsCard for themes', () => {
+  //   const userId = 123;
+  //   const { params, store } = signInUserWithUserId(userId);
+
+  //   renderUserProfile({ params, store });
+
+  //   expect(screen.getByText('Themes by')).toBeInTheDocument();
+  //   const extension_card = screen.queryByClassName('AddonsByAuthorsCard').at(1);
+  //   expect(screen.getByText(extension_card, 'Previous')).toBeInTheDocument();
+  //   expect(screen.getByText(extension_card, 'Next')).toBeInTheDocument();
+  //   expect(
+  //     screen.getByText(extension_card, `/user/${userId}/?page_t=2`),
+  //   ).toBeInTheDocument();
+  // });
+
+  // it('renders a not found page if the API request is a 404', () => {
+  //   const { store } = dispatchSignInActions();
+  //   const errorHandler = new ErrorHandler({
+  //     id: 'some-error-handler-id',
+  //     dispatch: store.dispatch,
+  //   });
+  //   errorHandler.handle(
+  //     createApiError({
+  //       response: { status: 404 },
+  //       apiURL: 'https://some/api/endpoint',
+  //       jsonResponse: { message: 'not found' },
+  //     }),
+  //   );
+
+  //   renderUserProfile({ errorHandler, store });
+
+  //   expect(screen.getByText('Oops! We can’t find that page')).toHaveLength(1);
+  // });
+
+  // it('renders errors', () => {
+  //   const { store } = dispatchSignInActions();
+  //   const errorHandler = new ErrorHandler({
+  //     id: 'some-id',
+  //     dispatch: store.dispatch,
+  //   });
+  //   const errorString = 'unexpected error';
+  //   errorHandler.handle(new Error(errorString));
+
+  //   renderUserProfile({ errorHandler, store });
+
+  //   expect(screen.getByText(errorString)).toBeInTheDocument();
+  // });
 
   it('renders an edit link', () => {
-    const root = renderUserProfile();
+    renderUserProfile();
 
-    expect(root.find('.UserProfile-edit-link')).toHaveLength(1);
-    expect(root.find('.UserProfile-edit-link')).toHaveProp('to', `/users/edit`);
-    expect(root.find('.UserProfile-edit-link').children()).toHaveText(
-      'Edit profile',
+    const editButton = screen.getByRole('link', { name: 'Edit profile' });
+    expect(editButton).toBeInTheDocument();
+    expect(editButton).toHaveAttribute(
+      'href',
+      `/${lang}/${clientApp}/users/edit`,
```

Comment:
I tried it locally and it worked for me. Have you pushed your latest changes? If so I can take another look.

Reply:
Still failing for me - I'd made some further changes since I wrote the comment so I've pushed an up to date commit (https://github.com/mozilla/addons-frontend/commit/997c66496d6319b111276378b64f1261ae8a8f64).  This is the error reported:
```
Summary of all failing tests
 FAIL  tests/unit/amo/pages/TestUserProfile.js (66.343 s)
  ● /mnt/c/users/andrew Williamson/dev/addons-frontend/tests/unit/amo/pages/TestUserProfile.js › renders an edit link

    expect(element).toHaveAttribute("href", "/fr/firefox/users/edit") // element.getAttribute("href") === "/fr/firefox/users/edit"

    Expected the element to have attribute:
      href="/fr/firefox/users/edit"
    Received:
      href="/en-US/firefox/users/edit"

      486 |     const editButton = screen.getByRole('link', { name: 'Edit profile' });
      487 |     expect(editButton).toBeInTheDocument();
    > 488 |     expect(editButton).toHaveAttribute(
          |                        ^
      489 |       'href',
      490 |       `/${lang}/${clientApp}/users/edit`,
      491 |     );

      at Object.<anonymous> (tests/unit/amo/pages/TestUserProfile.js:488:24)
```

Reply:
circleci concurs: https://app.circleci.com/pipelines/github/mozilla/addons-frontend/6889/workflows/21cdc3ab-2045-49e9-8197-751959eeea11/jobs/48553?invite=true#step-105-191